torch/csrc/jit/tensorexpr/tensor.h

```diff
@@ -300,6 +307,20 @@ inline ExprHandle Placeholder::load(const std::vector<T>& args) const {
       new Load(data(), ExprHandleVectorToExprVector(params), new IntImm(1)));
 }
 
+template <typename... Ts>
+inline ExprHandle BufHandle::load(const Ts&... ts) const {
+  std::vector<ExprHandle> params({ExprHandle(ts)...});
+  return ExprHandle(
+      new Load(node(), ExprHandleVectorToExprVector(params), new IntImm(1)));
+}
+
+template <typename T>
+inline ExprHandle BufHandle::load(const std::vector<T>& args) const {
```

Comment:
Is there a real need to make this a template? I think all our cases would be served when `T=ExprHandle`

Reply:
Why are the other ones templates? I just copied what the other `load` definitions were doing.

Reply:
🤷‍♂️ I think it meant to support vectors of `ExprHandle` and vectors of `VarHandle`, but I to me from the semantics point of view var handles should only be used as ExprHandles anyway, so it's better to encode that in the function signature. It might require building exprhandle vectors from varhandle vectors in some places but I think 1) it's a correct thing to do, 2) we could add helper methods like `VarHandleVectorToExprHandleVector` if that becomes too repetetive.

But following an existing way of doing things is a valid argument. Maybe if we decide to change that we need to do it in one go everywhere.